src/main/scala/org/allenai/plugins/DeployPlugin.scala

```diff
@@ -245,6 +245,12 @@ object DeployPlugin extends AutoPlugin {
     log.info("Deploy complete. Validate your server!")
   }
 
+  val stageAndChecksumTask = stageAndChecksum := {
+    val stageDir = (UniversalPlugin.autoImport.stage in thisProject).value
+    VersionInjectorPlugin.autoImport.injectCacheKey.value
```

Comment:
This still doesn't fix the problem that the resource is generated in the `resources` folder, but not bundled into the jar.